lib/watcher/heart-beat.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+exports.worker = function(handle) {
+  var intervalObj = null;
+  var _heartBeatInterval = 15 * 1000;
+  intervalObj = setInterval(function() {
+    handle.send({
+      cmd: 'metrics',
+      record: JSON.stringify({dummyRecord: ''}),
+    });
+  }, _heartBeatInterval);
+
+  this.stopWorker = function() {
```

Comment:
The difference is ```var intervalObj``` and ```this.stopWorke``` will be in a same closure with this code. with exports I have to make the variable global to be accessible in stop() method.